packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -209,31 +185,58 @@ events.
     }
   });
 </script>
-```
+``` 
+:::
 
-In this example we are subscribing to `UserAcceptedAQuery` and `UserClickedResultAddToCart` so in
-this way we can perform an action inside our application when these events are triggered. Even
-though we have subscribed to two events, as Interface X Components is an event based architecture
-there are more than one hundred of events available to be subscribed to. The complete list is not
-yet part of this documentation, but you may access to the
-[XEventsTypes](https://github.com/empathyco/x/blob/main/packages/x-components/src/wiring/events.types.ts)
-where all of them are defined.
```

Comment:
Why are we removing this explanation? I know there are things that you are telling ion other side, but not all of them.

Reply:
I think I've removed nothing, but rewritten or replaced information. Now, callout information is now available as a stand-alone subsection covering the example and the info related to X events as well.